packages/zk-eddsa-event-ticket-pcd/src/ZKEdDSAEventTicketPCD.ts

```diff
@@ -380,15 +376,6 @@ function claimFromProofResult(
   if (!babyJubIsNegativeOne(publicSignals[8])) {
     partialTicket.ticketCategory = parseInt(publicSignals[8]);
   }
```

Comment:
Is suggest checking that these public signal fields are zero and throwing if they're not.